src/Symfony/Component/HttpFoundation/Response.php

```diff
@@ -369,6 +369,10 @@ public function sendContent(): static
      */
     public function send(): static
     {
+        if ('' !== $this->content && \is_string($this->content) && !$this->headers->has('Transfer-Encoding')) {
```

Comment:
```suggestion
        if (\is_string($this->content) && '' !== $this->content && !$this->headers->has('Transfer-Encoding')) {
```

what bout this for readability? check string then not empty by order :)

Reply:
works for me, updated